doc/api/fs.md

```diff
@@ -3598,6 +3598,53 @@ that represent files will be deleted. The permissive behavior of the
 `recursive` option is deprecated, `ENOTDIR` and `ENOENT` will be thrown in
 the future.
 
+## `fs.rm(path[, options], callback)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* `options` {Object}
+  * `force` don't error on nonexistent path
+  * `maxRetries` {integer} If an `EBUSY`, `EMFILE`, `ENFILE`, `ENOTEMPTY`, or
+    `EPERM` error is encountered, Node.js will retry the operation with a linear
+    backoff wait of `retryDelay` ms longer on each try. This option represents
```

Comment:
from the sidelines: any particular reason this uses the abbreviated `ms` rather than typing out `milliseconds`? I see the `rm*` methods in these docs uses `ms`, but they are the only ones - all others use the full `milliseconds`

Reply:
No specific reason. This particular text was just copied from recursive rmdir. I can change it here or do it in my follow up PR to add the deprecation warning to recursive rmdir.